src/index.js

```diff
@@ -43,7 +38,7 @@ enableSlashCommands(state);
  * Starts the bot.
  */
 (async () => {
-    await app.start(process.env.PORT || 3000);
```

Comment:
https://github.com/slackapi/bolt-js/issues/1200 on kyl huonosti dokumentoitu, mut ainakaan `ss -lntp` mukaan tää ei taida nyt deployatussa muodossa tosiaankaan kuunnella mitään, eikä toi port 3000 ole varaman aiemminkaan tehnyt mitään, kun `socketMode: true` käytössä.

Reply:
Joo tää oli vähän sekava, mutta tarvitaan siinä tapauksessa jos appi käyttää OAuthia: https://slack.dev/bolt-js/tutorial/getting-started#listening-and-responding-to-a-message

Reply:
Käytetäänkö me OAuthia, vai onko se ainoastaan silloin jos tarjoaa appia muille asennettavaksi?

Reply:
Se on joo sillon jos tarjotaan tätä appia muille. Ei siis ymmärtääkseni oo meillä käytössä. Bottikäyttäjälle ollaan määritelty OAuth scopet ja se käyttää tunnistautuminen OAuthia ilmeisesti Slackissa sisäisesti kun se ottaa yhteyttä meidän workspaceen, mutta se on erillinen liity tähän.